kiva/agg/tests/test_stroke_path.py

```diff
@@ -237,47 +238,13 @@ def test_alias_cap_round_equality(self):
 
         # Instead of testing against a known desired value, we are simply
         # testing for equality...
-        self.assertRavelEqual(gc1.bmp_array[:,:,0], gc2.bmp_array[:,:,0])
-
-    def test_alias_cap_square(self):
```

Comment:
umm. why did black remove this test? :|

Reply:
_I_ Removed this test (and another in here) because it was a duplicate of another test